packages/modules/devices/huawei_smartlogger/config.py

```diff
@@ -0,0 +1,73 @@
+from typing import Optional
+from helpermodules.auto_str import auto_str
+from modules.common.component_setup import ComponentSetup
+
+
+@auto_str
+class Huawei_SmartloggerConfiguration:
+    def __init__(self, ip_address: Optional[str] = None):
+        self.ip_address = ip_address
```

Comment:
Ist im Smartlogger der Port einstellbar? Dann macht es Sinn, ihn auch hier in die Einstellungen aufzunehmen.